### ISSUE TITLE
Please bowerify angular-truncate.

### ISSUE THREAD

**Issue by trivektor** [sparkalow/angular-truncate#14]

**Comment by h2non** [comment]
Why? It's already "browerifed"

**Comment by goodbomb** [comment]
There doesn't seem to be a version when installing via bower. v0.1.0 that's listed in the bower.json file doesn't resolve when attempting to install it. Would be good to add proper versioning. Cheers!

**Comment by jmcclell** [comment]
Agreed. Bower install does not work when trying to specify version.

**Comment by peternixey** [comment]
+1 :+1:

**Comment by chazzlabs** [comment]
:+1:

**Comment by focusaurus** [comment]
:+1:

**Comment by juristr** [comment]
:+1:

**Comment by darylyu** [comment]
:+1: 

It's weird using master as the version.

**Comment by BernardoSilva** [comment]
Started [supported version of this repo](https://github.com/BernardoSilva/angular-truncate) to add proper versioning and keep it maintained.
It may help if you want to have all your dependencies specified like I need. :)

Raise issues or contribute if you want. :+1:
